src/libraries/shims/manual/System.Data.csproj

```diff
@@ -2,4 +2,9 @@
   <ItemGroup>
     <Compile Include="System.Data.forwards.cs" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup>
+    <!-- System.Data.SqlClient is not live built anymore, make sure to use reference assembly for it -->
+    <ReferencePath Remove="$(RefPath)\System.Data.SqlClient.dll" />
+    <ReferencePath Include="$(ArtifactsBinDir)ref\$(NetCoreAppCurrent)\System.Data.SqlClient.dll" />
```

Comment:
Nit: why not use [RefPath](https://github.com/dotnet/runtime/blob/95b361d7b9551e59f679ecd75cfc1480ee4e9c59/src/libraries/Directory.Build.props#L252) here as well?

Reply:
Because of it is re-defined to mean something else [here](https://github.com/dotnet/runtime/blob/master/src/libraries/shims/manual/Directory.Build.props#L15).